src/Simulation/QsharpCore/Microsoft.Quantum.QSharp.Core.csproj

```diff
@@ -1,9 +1,13 @@
-﻿<Project Sdk="Microsoft.Quantum.Sdk/0.13.2011.802-alpha">
+﻿<Project Sdk="Microsoft.Quantum.Sdk/0.14.2012122063-beta">
 
   <Import Project="..\TargetDefinitions\TargetPackages\QsharpCore.Package.props" />
 
   <PropertyGroup>
     <Description>Core support for the Q# programming language.</Description>
+    <GenerateConcreteIntrinsic>true</GenerateConcreteIntrinsic>
```

Comment:
Actually, that name would imply something other than what is happening here. These changes do not include automatic generation of the interfaces, those are still written by hand (see response blow about needing to check in those manual files). This does include the optional flag that lets a Q# project opt into the interface design such that the `body intrinsic` Q# declarations will produce concrete classes that call into the interfaces when generating C# (instead of abstract classes that must be overridden by an inheritor). This behavior has to be opt-in because otherwise it would be a breaking change. This way, any external developers writing their own `body intrinsic` operations can continue to use the old method while new projects like the target packages can use the new infrastructure.